README.md

```diff
@@ -1,5 +1,6 @@
 # UBCRocketGroundStation 
 [![Build Status](https://circleci.com/gh/UBC-Rocket/UBCRocketGroundStation.svg?style=shield)](https://app.circleci.com/pipelines/github/UBC-Rocket/UBCRocketGroundStation)
+[![codecov](https://codecov.io/gh/UBC-Rocket/UBCRocketGroundStation/branch/master/graph/badge.svg?token=2IML1026UZ)](https://codecov.io/gh/UBC-Rocket/UBCRocketGroundStation)
```

Comment:
🔥 👍 🚀 💯

Reply:
Is this badge showing integration or unit test coverage? Is there a way to choose?

Reply:
I think just overall: https://docs.codecov.com/docs/status-badges

Reply:
Overall is good!